src/app/images/components/SelectUpstreamImagesForm/_index.scss

```diff
@@ -1,4 +1,12 @@
 @mixin DownloadImages {
+  .multi-select div {
+    background-color: var(--vf-color-background-default);
+  }
+
+  .multi-select__select-button[aria-expanded=true] {
+    background-color: var(--vf-color-background-default) !important;
+  }
+
```

Comment:
Could you put these in the vanilla overrides file? In case we end up using MultiSelect in other places